pysmt/factory.py

```diff
@@ -416,10 +416,14 @@ class field or if a logic subsuming the given logic is
     ##
     ## Wrappers: These functions are exported in shortcuts
     ##
-    def Solver(self, quantified=False, name=None, logic=None):
+    def Solver(self, quantified=False, name=None, logic=None, options=None):
```

Comment:
Maybe we can allow the use of `**kwargs` to specify something like:
```python
Solver(logic=logic, incremental=False)
```
instead of

```python
Solver(logic=logic, options={"incremental": False})
```

??